content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -239,16 +236,18 @@ sudo mkdir -p "$DOWNLOAD_DIR"
 Install crictl (required for kubeadm / Kubelet Container Runtime Interface (CRI))
 
 ```bash
-CRICTL_VERSION="v1.26.0"
+CRICTL_VERSION="v1.27.0"
```

Comment:
Would this work?
```suggestion
CRICTL_VERSION="v{{< skew currentPatchVersion >}}"
```

Reply:
Em... it depends on whether crictl follows all patch versions of Kubernetes.

Reply:
i don't think crictl is guaranteed to match the k8s releases. there could be drift, thus a period of time when this script does not work.